homeassistant/components/tuneblade_remote/types.py

```diff
@@ -0,0 +1,14 @@
+"""Type definitions for the TuneBlade Remote integration.
+
+This module defines runtime data structures used by the TuneBlade Remote integration.
+"""
+
+from typing import TypedDict
+
+from .coordinator import TuneBladeDataUpdateCoordinator
+
+
+class TuneBladeRuntimeData(TypedDict):
+    """Runtime data stored for the TuneBlade Remote integration."""
+
+    coordinator: TuneBladeDataUpdateCoordinator
```

Comment:
we don't need to create a specific class if we are only going to store one thing in there